move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -141,6 +141,7 @@ void move_group::MoveGroupMoveAction::executeMoveCallback_PlanOnly(const moveit_
 {
   ROS_INFO("Planning request received for MoveGroup action. Forwarding to planning pipeline.");
 
+  context_->planning_scene_monitor_->syncSceneUpdates();
```

Comment:
If only the robot state would be considered, yes.
However, I want to ensure that the whole scene with collision objects etc. is up-to-date.